modules/get-package.xq

```diff
@@ -52,7 +54,13 @@ return
         let $xar := util:binary-doc($config:packages-col || "/" || $xar-filename)
         let $log := local:log-get-package-event($xar-filename)
         return
-            response:stream-binary($xar, "application/zip")
+            if ($wants-zip) then
+                let $entry := <entry type="binary" method="store" name="/{$xar-filename}" strip-prefix="false">{$xar}</entry>
```

Comment:
I believe `strip-prefix` is an argument to the `compression:zip` function, not an attribute on the `<entry>` element. See https://exist-db.org/exist/apps/fundocs/view.html?uri=http://exist-db.org/xquery/compression#zip.3.

Reply:
I see that was the way the [original code](https://github.com/eXist-db/public-repo/commit/da8d56151dab330e9ca52e05fe17dbfee17e7a2a#diff-bc346eefb44d6303a883aca1f15e3f0e68895c005bfa330f2ea00b7bf2cc70f0) was written - so presumably it was harmless. Do you agree? If so, I think we could remove it here or in a future PR.

Reply:
@joewiz I just checked the Java code behind `compress:zip`, indeed the `strip-prefix` is not used. So I have pushed a commit which removes that.